docs/basic-usage.md

```diff
@@ -83,7 +83,7 @@ python = "^3.7.0"
 
 we are allowing any version of Python 3 that is greater than `3.7.0`.
 
-When you run `poetry install`, you must have access to some version of a Python inrepreter that satisfies this constraint available on your system.
+When you run `poetry install`, you must have access to some version of a Python intepreter that satisfies this constraint available on your system.
```

Comment:
```suggestion
When you run `poetry install`, you must have access to some version of a Python interpreter that satisfies this constraint available on your system.
```
😉

Reply:
OMG, thanks! I had to force push twice in order to fix my own mistake!